verification/test-wdls/TestUltimaGenomicsWholeGenomeCramOnly.wdl

```diff
@@ -0,0 +1,134 @@
+version 1.0
+
+
+import "../../pipelines/broad/dna_seq/somatic/single_sample/ugwgs/UltimaGenomicsWholeGenomeCramOnly.wdl" as UltimaGenomicsWholeGenomeCramOnly
+import "../../verification/VerifyUltimaGenomicsWholeGenomeCramOnly.wdl" as VerifyUltimaGenomicsWholeGenomeCramOnly
+import "../../tasks/broad/Utilities.wdl" as Utilities
+import "../../tasks/broad/CopyFilesFromCloudToCloud.wdl" as Copy
+
+workflow TestUltimaGenomicsWholeGenomeCramOnly {
+
+    input {
+      ContaminationSites contamination_sites
+      AlignmentReferences alignment_references
+      VcfPostProcessing vcf_post_processing
+      Array[File]? input_cram_list
+      Array[File]? input_bam_list
+      String base_file_name
+      Float rsq_threshold = 1.0
+      Int reads_per_split = 20000000
+      Boolean save_bam_file = false
+
+      # These values will be determined and injected into the inputs by the scala test framework
+      String truth_path
+      String results_path
+      Boolean update_truth
+      String vault_token_path
+      String google_account_vault_path
+    }
+
+    meta {
+      allowNestedInputs: true
+    }
+  
+    call UltimaGenomicsWholeGenomeCramOnly.UltimaGenomicsWholeGenomeCramOnly {
+      input:
+        contamination_sites = contamination_sites,
+        alignment_references = alignment_references,
+        vcf_post_processing = vcf_post_processing,
+        input_cram_list = input_cram_list,
+        input_bam_list = input_bam_list,
+        base_file_name = base_file_name,
+        rsq_threshold = rsq_threshold,
+        reads_per_split = reads_per_split,
+        save_bam_file = save_bam_file
+  
+    }
+
+    
+    # Collect all of the pipeline outputs into single Array[String]
+    Array[String] pipeline_outputs = flatten([
+                                    [ # File outputs
+                                    UltimaGenomicsWholeGenomeCramOnly.agg_quality_distribution_pdf,
+                                    UltimaGenomicsWholeGenomeCramOnly.agg_gc_bias_pdf,
+                                    UltimaGenomicsWholeGenomeCramOnly.selfSM,
+                                    UltimaGenomicsWholeGenomeCramOnly.output_cram_md5,
+                                    UltimaGenomicsWholeGenomeCramOnly.output_cram_index,
+                                    UltimaGenomicsWholeGenomeCramOnly.output_cram,
+                                    ],
+                                    # File? outputs
+                                    select_all([UltimaGenomicsWholeGenomeCramOnly.output_bam_index]),
```

Comment:
Hopefully in all cases for `UltimaGenomicsWholeGenomeCramOnly` these output bam and output bam_index files should always be null. Not sure if that's worth adding a test for?

Reply:
@meganshand if these are always null i can just remove them from this pipeline_outputs array. Do you want me to add a check somewhere to see if the bams are null and fail if they are not null?

Reply:
Hi @meganshand, just making sure you saw the comment above!